src/components/MoneyReportHeader.tsx

```diff
@@ -230,7 +232,8 @@ function MoneyReportHeader({policy, report: moneyRequestReport, transactionThrea
                             shouldDisableApproveButton={shouldDisableApproveButton}
                             style={[styles.pv2]}
                             formattedAmount={!ReportUtils.hasOnlyHeldExpenses(moneyRequestReport.reportID) ? displayedAmount : ''}
-                            isDisabled={!canAllowSettlement}
+                            isDisabled={isOffline && !canAllowSettlement}
```

Comment:
Should we apply the same logic to `SettlementButton` [here](https://github.com/Expensify/App/blob/734ae64a85998f92c02ec959511f48385bef26ea/src/components/MoneyReportHeader.tsx#L307)?

Reply:
Great catch! I've added it